cmd/aries-agentd/startcmd/webhook/webhook.go

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
@sudeshrshetty @DRK3 Is this package intended to be not re-usable? (it's in the cmd/ path rather than the pkg/ path).

Reply:
Moved it to pfk/restapi/webhook.